builder/azure/common/client/azure_client_set.go

```diff
@@ -50,12 +45,18 @@ type AzureClientSet interface {
 var _ AzureClientSet = &azureClientSet{}
 
 type azureClientSet struct {
```

Comment:
While this client is in the common package it is only used by the chroot builder, the way I made this work with the new clients is a bit hacky, but I felt it wasn't worth a lot of time refactoring the chroot builder heavily as it makes up a very small percent of our usage.